Snakefile

```diff
@@ -88,6 +117,16 @@ rule get_merizo_predictions:
             > {output.output_file}
         """
 
+rule get_mdtask_clustering:
+    input:
+        correlation_file="output/correlation/{pdb_file}.txt"
+    output:
+        output_file="output/correlation/{pdb_file}.tsv"
+    shell:
+        """
+        python scripts/segment_by_correlation.py {input.correlation_file} {output.output_file}
+        """
```

Comment:
The correlation file is just a txt with a matrix of numbers from 0 to 1. The png shows a heatmap.